server/games/game.py

```diff
@@ -80,6 +80,8 @@ class ValidityState(IntEnum):
     FFA_NOT_RANKED = 14
     UNEVEN_TEAMS_NOT_RANKED = 15
     UNKNOWN_RESULT = 16
+    MULTI_TEAM = 17
+    HAS_AI = 18
```

Comment:
Both are not self explanatory, please extend the name (like UNEVEN_TEAM_NOT_RANKED) or at least add a comment

Reply:
MULTI_TEAM is already referenced in the DB so i'm going to comment this one.
I might be able to change HAS_AI though

Reply:
Those are already documented in the DB with a fancy comment and referenced under the exact same name - they're also in the category "ValidityState", making their function quite obvious